packages/auto-approve/src/get-PR-info.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// eslint-disable-next-line node/no-extraneous-import
+import {ProbotOctokit} from 'probot';
+
+// This file gets information about the incoming pull request, such as what files were changed, etc.
+
+export interface File {
+  sha: string;
+  filename: string;
+}
+/**
+ * Returns file names of the PR that were changed.
+ *
+ * @param octokit Octokit instance
+ * @param owner string, the owner of the repo
+ * @param repo string, the name of the repo
+ * @param prNumber number, the number of the repo
+ * @returns an array of File objects that were changed in a pull request
+ */
+export async function getChangedFiles(
+  octokit: InstanceType<typeof ProbotOctokit>,
+  owner: string,
+  repo: string,
+  prNumber: number
+): Promise<File[]> {
+  return (
+    await octokit.pulls.listFiles({
+      owner,
+      repo,
+      pull_number: prNumber,
+    })
+  ).data;
+}
+
+/**
+ * Gets the blob of a file from an array of changed files from a PR
```

Comment:
Thank you for writing per-function comments like this.  They help the reader (me) so much.

Reply:
Thanks for appreciating :)